src/libraries/Fuzzing/DotnetFuzzing/Fuzzers/TextEncodingFuzzer.cs

```diff
@@ -0,0 +1,191 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#define FORWARD_TO_NETFRAMEWORK
+
+using System.Buffers;
+using System.Diagnostics;
+using System.Text;
+
+namespace DotnetFuzzing.Fuzzers;
+
+// The fuzzing infrastructure currently does not support fuzzing .NET Framework.
+// However, this test class, while running under .NET Core, was used to foward the fuzzing
+// input to a .NET Framework console app. That app had the same test semantics as the tests
+// here, although used slightly different supporting APIs since not all supporting library
+// and language features are present in .NET Framework.
```

Comment:
how this is working? I am not seeing any forwarding to console app here? is it done somewhere else when you do `#define FORWARD_TO_NETFRAMEWORK`?

Reply:
The code to do this was removed from this PR for a few reasons, but exists in the commit history. See https://github.com/dotnet/runtime/pull/103968/commits/94010d8edda6cf4d12bcde4ee7fac440c33c1f21#diff-1249360cc8e4b2057cb1111a705788d87911917d770743eae1779847159f9790R66-R67